src/main/resources/assets/appliedenergistics2/lang/de_DE.lang

```diff
@@ -50,12 +51,36 @@ tile.appliedenergistics2.BlockCraftingAccelerator.name=Fertigungs-Prozessoreinhe
 tile.appliedenergistics2.BlockCraftingUnit.name=Fertigungseinheit
 tile.appliedenergistics2.BlockMolecularAssembler.name=Molekularassembler
 
+// Stairs
+tile.appliedenergistics2.ChiseledQuartzStairBlock.name=Gemeißelte Certusquartztreppe
+tile.appliedenergistics2.FluixStairBlock.name=Fluixtreppe
+tile.appliedenergistics2.QuartzPillarStairBlock.name=Certusquartzsäulentreppe
+tile.appliedenergistics2.QuartzStairBlock.name=Certusquartztreppe
+tile.appliedenergistics2.SkyStoneBlockStairBlock.name=Himmelssteinblocktreppe
+tile.appliedenergistics2.SkyStoneBrickStairBlock.name=Himmelssteinziegeltreppe
+tile.appliedenergistics2.SkyStoneSmallBrickStairBlock.name=Kleine Himmelssteinziegeltreppe
```

Comment:
Wird das in anderen Mods auch so übersetzt? Hier ist ja nicht die Treppe klein, sondern die Ziegeln